### PR TITLE
Update libnfc to 1.7.1

### PR DESCRIPTION
**Pull request by neffs**

### CODE REVIEW

File: Library/Formula/libnfc.rb [hunk 1]
```diff
@@ -31,14 +32,27 @@ def install
 end
 
 __END__
+diff --git a/libnfc/buses/uart.c b/libnfc/buses/uart.c
```
Comment (mikemcquaid) [inline Library/Formula/libnfc.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

Comment (neffs) [inline Library/Formula/libnfc.rb]: It was in the previous version, I just changed the patch for 1.7.1. Maybe some USB-serial adapter which the previous author has uses one of these device names.

Comment (mikemcquaid) [inline Library/Formula/libnfc.rb]: I'd still like it if the above could be done anyway, thanks.

**Comment by neffs** [comment]
https://github.com/nfc-tools/libnfc/pull/290

File: Library/Formula/libnfc.rb [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 class Libnfc < Formula
   homepage 'http://www.libnfc.org/'
-  url 'https://libnfc.googlecode.com/files/libnfc-1.7.0.tar.bz2'
-  sha1 '5adfb6c6238b1659ad8609837dc8e59eb41a8768'
+  url 'https://bintray.com/artifact/download/nfc-tools/sources/libnfc-1.7.1.tar.bz2'
+  sha1 'd77c1408f5e38f24d69778b2ff82f68b2d7bc111'
+  head 'https://code.google.com/p/libnfc/'
```
Comment (asparagui) [inline Library/Formula/libnfc.rb]: this seems to go to https://github.com/nfc-tools/libnfc

File: Library/Formula/libnfc.rb (hunk 1 above)
Comment (asparagui) [inline Library/Formula/libnfc.rb]: This patch can be replaced with: https://github.com/nfc-tools/libnfc/commit/1785ac14c2eab7e66877456845dff29e8b51cb94.diff

Comment (mikemcquaid) [inline Library/Formula/libnfc.rb]: Yeh, that'd be preferable.

Comment (neffs) [inline Library/Formula/libnfc.rb]: done

Comment (mikemcquaid) [inline Library/Formula/libnfc.rb]: Can you use a `patch do` block with that URL?

**Comment by DomT4** [comment]
Can you fixup the comments highlighted by Brett and Mike in this PR? Thanks!

**Comment by DomT4** [comment]
Closing here. If you're still interested feel free to address other maintainer comments and I'll happily reopen.

**Pull request closed without merging**